docs/.sphinx/pa11y.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Do we still need this file?

Reply:
Probably not, I checked, it's only used in the CI which we removed. I will remove this file and make sure the build succeeds.